tests/_files/BankAccount-lcov-line.info

```diff
@@ -0,0 +1,27 @@
+TN:
+SF:/app/tests/_files/BankAccount.php
```

Comment:
`SF:%s/tests/_files/BankAccount.php` should get the CI passing 🤞

Reply:
opps, for to adjust them :facepalm:

Reply:
Thats weird, it worked before with  `%s%e`, I was following whats in the `BankAccount-clover-line.xml` :thinking: